src/main/java/seedu/address/model/calendar/MeetingTimeComparator.java

```diff
@@ -0,0 +1,34 @@
+package seedu.address.model.calendar;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Comparator;
+import java.util.Date;
+
+public class MeetingTimeComparator extends MeetingComparator implements Comparator<CalendarMeeting> {
+
+    public static final String SORT_CRITERIA = "time";
+    public static final String MESSAGE_INVALID_TIME = "Time is not in valid format.";
+
+    @Override
+    public int compare(CalendarMeeting meeting1, CalendarMeeting meeting2) {
+        String time1 = meeting1.getCalendarTime().time;
+        String time2 = meeting2.getCalendarTime().time;
```

Comment:
Maybe it would be better to store a Date object in the meetings rather then a string so that you woudnt have to keep parsing everywhere?